check_borg

```diff
@@ -20,16 +22,25 @@ else
 	exit 128
 fi
 
-debug(){   [ "$verbose" -gt 1 ] && echo "$*"; }
-verbose(){ [ "$verbose" -gt 0 ] && echo "$*"; }
+debug(){
```

Comment:
I see the reasoning of your if/else construct. But here I'd prefer properly associated `&&||`. `([ "$verbose" -gt 1 ] && echo "$*") || return 0`